README.md

```diff
@@ -89,6 +89,9 @@ Participating discussions via the following IM groups is also welcomed.
 Over the past few years, NNI has received thousands of feedbacks on GitHub issues, and pull requests from hundreds of contributors.
 We appreciate all contributions from community to make NNI thrive.
 
+<!-- contributors-NUMBER-orange NUMBER should be rewrite by hand -->
```

Comment:
You should add this on the top of the page, in the badge list?

Reply:
![](https://img.shields.io/github/contributors-anon/microsoft/nni)    +1

// https://img.shields.io/github/contributors-anon/microsoft/nni